GENETIS_doc/.gitignore

```
@@ -0,0 +1,3 @@
+.venv/
+*DS_Store
+docs/.vscode/
```

```suggestion
docs/.vscode/
docs/build/
```
The `build` directory should be ignored, since it's generated.

---

I thought Vercel needs access to this directory?

---

I'm not familiar with Vercel, but generally you don't want to include the `build` dir but let whatever tool you are using rebuild the latest version of the docs.  Readthedocs works like that.  If you can't tell Vercel to install sphinx-build as a dependency, maybe you could try readthedocs instead.

---

Thanks for the tips! Alan showed me Jupyter-book + Github Pages (and I like that combo a bit more) which also allows for ignoring the `_build` directory.